controllers/ads_controller.rb

```diff
@@ -0,0 +1,24 @@
+get '/ads' do
+  ads = Ad.order(:created_at).limit(100)
```

Comment:
will add pagination as well, turned out it's not that simple to add a scope with Sequel